test/model-method.js

```diff
@@ -105,7 +105,10 @@ describe('ModelMethod', function() {
 describe('ModelMethod - Loopback 2.0', function() {
   var userModel;
 
-  beforeEach(givenLB2Workspace);
```

Comment:
Could you please preserve `givenLB2Workspace` and `givenLB3Workspace` helpers? IMO it makes the code easier to read by hiding implementation details that are not important.